suite/tests/CMakeLists.txt

```diff
@@ -2121,6 +2121,16 @@ if (CLIENT_INTERFACE)
       "" "")
   endif (NOT ARM)
 
+  add_library(client.blackbox.privdll SHARED client-interface/blackbox.privdll.c)
```

Comment:
I find the name confusing: it implies that blackbox.dll.c is *not* private.  blackbox.dll{1,2} or blackbox.dll{A,B} or sthg seems clearer.